lib/web/scripts/install.go

```diff
@@ -178,7 +180,22 @@ var legacyInstallScript string
 // or by unpacking the tarball. Its usage should be phased out in favor of the updater-based
 // installation script served by getUpdaterInstallScript.
 func getLegacyInstallScript(ctx context.Context, opts InstallScriptOptions) (string, error) {
-	return legacyInstallScript, nil
+	tunedScript := strings.Replace(legacyInstallScript, `TELEPORT_VERSION=""`, fmt.Sprintf(`TELEPORT_VERSION="%s"`, opts.TeleportVersion), 1)
+	if opts.TeleportFlavor == types.PackageNameEnt {
+		tunedScript = strings.Replace(tunedScript, `TELEPORT_SUFFIX=""`, `TELEPORT_SUFFIX="-ent"`, 1)
+	}
+
+	var edition string
+	if opts.AutoupdateStyle == PackageManagerAutoupdate {
+		edition = "cloud"
```

Comment:
Is this a valid edition? We don't look for it in teleport-update

Reply:
The install script has its own concept of edition which is not the same than the modules, editions are `oss`, `enterprise`, and `cloud`.